exercises/D-logical-operators/exercise2.js

```diff
@@ -5,7 +5,36 @@
   Update the code so that you get the expected result.
 */
 
-function isNegative() {}
+function isNegative(number) {
+  if (number < 0) {
+    return true;
+  } else {
+    return false;
+  }
+}
+
+function isBetween5and10(num) {
+  if (num >= 5 && num <= 10){
+    return true;
+  } else {
+    return false;
+  }
+}
+
+function isShortName(name) {
+  if (name === "Daniel"){
+    return true;
+  } else {
+    return false;
+  }
+  
+}
+
+function startsWithD(nameStartWithD) {
+   
+    return nameStartWithD.startsWith("D");
+  
+}
```

Comment:
In this function you have used just one line. In the other methods you have done 2 separate checks. Both  are correct. Which do you prefer? Or does it depend? There's no right answer, but it's good to think about.

Reply:
Thanks Karen. Predicate function takes one value as input and return it true or false. I've changed the code to simplified it into one line. I think this will be good practice. Thanks for the feedback. Its helps me to improvise my code.